tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -598,6 +726,12 @@ def _call_openai_llm(OpenAI):
         with get_request_vcr(subdirectory_name="langchain_community").use_cassette("openai_completion_sync.yaml"):
             llm.invoke("Can you explain what Descartes meant by 'I think, therefore I am'?")
 
+    @staticmethod
+    def _call_openai_embedding(OpenAIEmbeddings):
+        embedding = OpenAIEmbeddings()
+        with get_request_vcr(subdirectory_name="langchain_community").use_cassette("openai_embedding_query.yaml"):
```

Comment:
```suggestion
        with mock.patch("langchain_openai.OpenAIEmbeddings._get_len_safe_embeddings", return_value=[0.0] * 1536):
            with get_request_vcr(subdirectory_name="langchain_community").use_cassette("openai_embedding_query.yaml"):
```
Since we're using OpenAIEmbeddings we need to mock out that internal tokenizer method that I mentioned earlier in the other tests as well 😄